<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>miniScopesContainer</class>
 <widget class="QWidget" name="miniScopesContainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>709</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>963</width>
    <height>688</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Osilloscopes Club</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="OscilloscopeMiniWindow" name="scopeWidget1" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>354</width>
       <height>225</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="OscilloscopeMiniWindow" name="scopeWidget4" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>354</width>
       <height>225</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="OscilloscopeMiniWindow" name="scopeWidget7" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>354</width>
       <height>225</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="OscilloscopeMiniWindow" name="scopeWidget2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>354</width>
       <height>225</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="OscilloscopeMiniWindow" name="scopeWidget5" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>354</width>
       <height>225</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="OscilloscopeMiniWindow" name="scopeWidget8" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>354</width>
       <height>225</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="OscilloscopeMiniWindow" name="scopeWidget3" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>354</width>
       <height>225</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="OscilloscopeMiniWindow" name="scopeWidget6" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>354</width>
       <height>225</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Quick Functions</string>
     </property>
     <widget class="QPushButton" name="ResetAllButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset All</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LockAllButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Lock All</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OscilloscopeMiniWindow</class>
   <extends>QWidget</extends>
   <header>OscilloscopeMiniWindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
